hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/util/Preconditions.java

```diff
@@ -249,4 +336,9 @@ static String getDefaultNullMSG() {
   static String getDefaultCheckArgumentMSG() {
     return CHECK_ARGUMENT_EX_MESSAGE;
   }
+
+  /* @VisibleForTesting */
```

Comment:
Thanks for this, I think we can catchup on this later based on which PR goes first.